src/components/TextField/TextField.tsx

```diff
@@ -1,13 +1,13 @@
 import clsx from 'clsx';
-import React, { ChangeEventHandler, ReactNode } from 'react';
+import React, { ChangeEventHandler, forwardRef, ReactNode } from 'react';
 import { useUID } from 'react-uid';
 import styles from './TextField.module.css';
 import FieldNote from '../FieldNote';
 import InputField from '../InputField';
 import Label from '../Label';
 import Text from '../Text';
 
-export interface Props {
+export type Props = React.InputHTMLAttributes<HTMLInputElement> & {
```

Comment:
1/2 changes besides forwarding the ref in this file is extending the HTMLInputProps to attach to the input element. Was already being done via `{...other}` but this would show what's available